features/bootstrap/FeatureContext.php

```diff
@@ -13,6 +13,8 @@
 use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\CompositeLabel;
 use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\CompositeRelation;
 use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\Dummy;
+use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\EmbeddableDummy;
+use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\EmbeddedDummy;
```

Comment:
Please do not add new properties to existing entities :) It's better to have a dedicated entity for every class of functional tests.
And they are just... tests. We'll not stop merging feature because tests can be modernized :)